civiform_config.example.sh

```diff
@@ -15,7 +15,7 @@
 #################################################
 
 # REQUIRED
-# One of prod, staging, or dev.
+# One of prod, staging, or test.
```

Comment:
suggestion (blocking): I'm inclined to only put "staging" and "prod" as options here for now until we can debug "test" deploys. Tom was unable to deploy in "test" mode and continued to get the S3 could not create resource error.

Reply:
done